Automattic/wp-calypso
# E2E: Read chromedriver version from file

#### Changes proposed in this Pull Request

* Read the Chromedriver version from file instead of env var. This will make it easier to sync with wrapper repos. 

#### Testing instructions
* Ensure that proper version of Chromedriver is downloaded in the setup steps in CircleCI
